src/db.cr

```diff
@@ -0,0 +1,7 @@
+class DB
```

Comment:
Better use another name. crystal-db uses DB and it might be needed in the future